READ_BEFORE_UPDATE.md

```diff
@@ -1,5 +1,38 @@
 # Update Notes
 
+## Migrating from privacyIDEA 3.9.2 to eduMFA
+
+> [!CAUTION]
+>  * Make a backup of your existing installation! This backup should include the config files, the encryption keys and
+     the database.
+>  * Ensure that you use the latest supported privacyIDEA version 3.9.2 and then upgrade to eduMFA
+> 
+> These migration steps are applicable for the versions 1.0.0 to 1.5.1
+
+* Uninstall privacyIDEA and stop your nginx or `Apache2` or `nginx` service
+* Move/Copy all configurations from `/etc/privacyidea/` to `/etc/edumfa/`
+    * The config file needs to be renamed from `pi.cfg` to `edumfa.cfg`
+    * Rename/Replace `PI_` in the configuration file with `EDUMFA_` (e.g. `PI_ENCFILE` changes to `EDUMFA_ENCFILE`)
+    * Update the paths in the configuration file to `/etc/edumfa/`
+    * Update the log file path to `/var/log/edumfa/edumfa.log`
+* Install eduMFA (e.g. using Container, PIP or the `.deb` Package)
+    * When using the `.deb` package, make sure to not overwrite the existing configuration file
+    * When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), all following steps will be performed automatically
```

Comment:
I believe crontab or systemd services will not be updated by the .deb packages

Reply:
The packages adds a new cron, however systemd keeps unchanged. Maybe we should rather mention, what will be done automatically like: "When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will executed automatically. Additionally the web server configurtion and a cron will also be installed.

Reply:
```suggestion
    * When using the server `.deb` package (`edumfa-apache2`, `edumfa-nginx`), the database migration will be executed automatically. Additionally the web server configuration and a cron entry will be installed.
```